justfile

```diff
@@ -119,7 +119,10 @@ run *flags: (_target-installed target)
     cargo run {{ _target-option }} {{ flags }}
 
 test: (_target-installed target)
-    cargo nextest run {{ _target-option }} --all-features
+    cargo nextest run {{ _target-option }} --all-features --target-dir target/tests
+
+test-package package *flags:
+    cargo nextest run --all-features --no-capture --package {{ package }} --target-dir target/tests {{ flags }}
```

Comment:
Unrelated sneaky change - I still want to update the dev docs but this is very useful in the mean time.